WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsSelectPlugin.swift

```diff
@@ -0,0 +1,99 @@
+import SwiftUI
+import Yosemite
+
+struct InPersonPaymentsSelectPluginRow: View {
+    let icon: UIImage
+    let name: String
+    let selected: Bool
+
+    var body: some View {
+        HStack(spacing: 16) {
+            Image(uiImage: icon)
+            Text(name)
+                .headlineStyle()
+                .fixedSize(horizontal: false, vertical: true)
+            Spacer()
+            if selected {
+                Image(systemName: "checkmark")
+                    .foregroundColor(Color(.primary))
+            } else {
+                Color.clear.frame(width: 24, height: 24, alignment: .center)
+            }
+        }
+        .padding(16)
+        .frame(maxWidth: .infinity, minHeight: 72, alignment: .leading)
+        .overlay(
+            RoundedRectangle(cornerRadius: 8)
+                .stroke(borderColor, lineWidth: 1)
+               )
+    }
+
+    var borderColor: Color {
+        if selected {
+            return Color(.primary)
+        } else {
+            return Color(.tertiaryLabel)
+        }
+    }
+}
+
+struct InPersonPaymentsSelectPlugin: View {
```

Comment:
Of course not a blocker, you could consider adding the `View` postfix here (and/or in the filename)  as we discussed in p1651641104110959-slack-CGPNUU63E

Reply:
Sure, added that in 73b1454a0f7c7584d005d405f2e7f6de7097fa97, but will leave it for #7059 since I might end up with conflicts otherwise